app/helpers/proxmox_vm_interfaces_helper.rb

```diff
@@ -56,7 +56,8 @@ def interface_compute_attributes_typed_keys(type)
   end
 
   def interface_common_typed_keys(type)
-    [{ origin: 'id', dest: 'id' }, { origin: 'mac', dest: type == 'qemu' ? 'macaddr' : 'hwaddr' }]
+    mac = type == 'qemu' ? 'macaddr' : 'hwaddr'
+    [{ origin: 'id', dest: 'id' }, { origin: mac, dest: mac }]
```

Comment:
If `origin` and `dest` are the same in both cases, is this necessary at all?

Reply:
changed it to only add the keys as a list.